app/controllers/home_controller.rb

```diff
@@ -5,7 +5,8 @@ class HomeController < ApplicationController
 
   def index
     @comments = Comment.latest.limit(10).all
-    @versions = OntologyVersion.latest.where(state: 'done').limit(10).all
+    @versions = OntologyVersion.accessible_by(current_user).latest.
+      where(state: 'done').limit(10).all
```

Comment:
I think (not sure, though) that the conversion to an array with`.all` is unnecessary overhead.